lib/ddtrace.rb

```diff
@@ -36,11 +36,13 @@ def self.tracer
       module Datadog
         # Railtie class initializes
         class Railtie < Rails::Railtie
+          # add instrumentation middlewares
+          options = {}
+          config.app_middleware.insert_before(0, Datadog::Contrib::Rack::TraceMiddleware, options)
```

Comment:
this change ensures that the middleware is here; with the previous approach, applications may ignore the initializer resulting in wrong traces